test_data/config.toml

```diff
@@ -321,15 +321,20 @@ post_cmd = '''
     mv "$RESPONSE_FILE" "$env_dir/$(basename $RESPONSE_FILE)"
 '''
 
-[envs.hello]
-skip_if_output_exists = "envs/hello"
+[envs.build-noop]
+skip_if_output_exists = "envs/build-noop"
 pre_cmd = "flox init"
-cmd = "flox install hello"
+cmd = '''
+echo '
+version = 1
+[build]
+hello.command = "echo hello"
+' | flox edit -f -
+'''
```

Comment:
note nonblocking: I think our env generation was designed more with catalog responses in mind, which this doesn't use. So I think it's clunky to use env generation for a lockfile like this

Reply:
would you usggest manually creating lockfiles in this case?
looks like the other `envs.*` are at least similar enough -- i can pull out the inline manifest.toml into a file if that helps

Reply:
I don't think you need it, I just think at some point we'll want to clean up our generation for environments that don't use the catalog